src/main/java/oss/fosslight/service/impl/T2UserServiceImpl.java

```diff
@@ -64,608 +60,623 @@
-		properties.put(Context.SECURITY_CREDENTIALS, userPw);
-		
-		String[] attrIDs = { "cn", "mail" };
-		if(StringUtil.isEmpty(filter)) {
-			filter = "(cn=" + userId + ")";
-		}
-		
-		DirContext con = null;
-		SearchControls constraints = new SearchControls();
-		NamingEnumeration<SearchResult> m_ne = null;
-		
-		try {
-			con = new InitialDirContext(properties);
-			isAuthenticated = true;
-			
-			constraints.setSearchScope(SearchControls.SUBTREE_SCOPE);
-			
-			if(attrIDs != null) {
-				constraints.setReturningAttributes(attrIDs);
-			}
-			
-			String searchKey = StringUtil.avoidNull(CommonFunction.getProperty("ldap.search.key"), "");
-			m_ne = con.search(searchKey, filter, constraints);
-		} catch (NamingException e) {
-			log.error(e.getMessage(), e);
-		} finally {
-			if(con != null) {
-				try {
-					con.close();
-				} catch (NamingException e) {}
-			}
-		}
-		
-		try {
-			SearchResult sr = null;
-
-			while (m_ne.hasMoreElements()) {
-				sr = (SearchResult) m_ne.next();
-				if(sr != null) {
-					String email = (String) sr.getAttributes().get("mail").get();
-					
-					if(!StringUtil.isEmpty(email)) {
-						userInfo.put("EMAIL", email);
-					}
-				}
-			}
-		} catch (Exception e) {
-			log.error(e.getMessage(), e);
-		}
-		
-		return isAuthenticated;
-	}
-	
-	@Override
-	public List<T2Users> checkEmail(String email) {
-		return  userMapper.selectCheckEmail(email);
-	}
-
-	@Override
-	public List<T2Users> getUserListByName(String name) {
-		return userMapper.getUserListByName(name);
-	}
-
-	/**
-	 * Gets the login user info.
-	 *
-	 * @return the login user info
-	 */
-	@Override
-	public T2Users getLoginUserInfo() {
-		T2Users bean = new T2Users();
-		bean.setUserId(bean.getLoginUserName());
-		
-		return userMapper.getUser(bean);
-	}
-
-	@Override
-	public boolean isLeavedMember(String userId) {
-		T2Users param = new T2Users();
-		param.setUserId(userId);
-		T2Users bean = userMapper.getUser(param);
-		
-		return bean == null || CoConstDef.FLAG_NO.equals(bean.getUseYn());
-	}
-
-	/**
-	 * Check api user auth.
-	 * API 요청시 사용자 ID, 패스워드 인증 처리
-	 * 인증성공인 경우 사용자 정보를 반환한다.
-	 * 인증실패인 경우 결과를 return 하지 않고 Exception처리한다.
-	 *
-	 * @param id the id
-	 * @param password the password
-	 * @return the t 2 users
-	 */
-	@Override
-	public T2Users checkApiUserAuth(String _token) {
-		T2Users params = new T2Users();
-		params.setToken(_token);
-		params = getUser(params); // 등록된 token 여부 확인
-		
-		if(params == null) {
-			// 미등록 token
-			throw new CUserNotFoundException();
-		}
-		
-		// Token 인증
-		if(checkToken(params, _token)) { // 추출된 USER 정보로 동일한 token이 생성이 되는지 확인.
-            return getUserAndAuthorities(params);
-        } else {
-            throw new CSigninFailedException();
+    } catch (Exception e) {
+      log.error(e.getMessage(), e);
+    }
+
+    return result;
+  }
+
+  @Override
+  public int changEnabled(T2Users t2Users) {
+    String enabled = Objects.equals(t2Users.getEnabled(), "true") ? "1" : "2";

```

Comment:
I changed to use `Object.equals()` instead of `==` operator when comparing Strings.